client/app/templates/show-project.hbs

```diff
@@ -8,8 +8,13 @@
           {{else}}
             Project {{model.dcpName}}
           {{/if}}
-        </h1>
 
+          {{#if this.debug}}
+            <a href="https://nycdcppfs.crm9.dynamics.com/main.aspx?etn=dcp_project&id={{{model.dcpProjectid}}}&pagetype=entityrecord" target="_blank">
```

Comment:
This should be dependent on environment. We're usually developing locally, and working with data on UAT2. UAT2 data can be pretty different from prod data

Reply:
> This should be dependent on environment. We're ~usually~ always developing locally, and working with data on UAT2. UAT2 data can be pretty different from prod data

Sure, I can add that.